VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -104,6 +104,29 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         return '';
     }
 
+    getParticipantRowClasses(participant: any): string {
+        if (!this.isJohConsultation()) {
+            return 'participant-row';
+        }
+        if (!this.isLastJohFromGroupOrIndividualGroupMember(participant)) {
+            return 'participant-group-row';
+        } else {
+            return 'participant-row';
+        }
+    }
```

Comment:
I think you should have all of them be a participant-row and have this return true or false and set an optional class based on that.

Reply:
I will try that